index.html

```diff
@@ -3325,6 +3323,49 @@ <h2>
       <h2>
         Security Considerations
       </h2>
+      <section class="informative">
+        <h2 id="sec-enhancements">
+          Mechanisms to enhance security
+        </h2>
+        <p>
+          This specification includes a number of mechanisms to enhance the
+          security of the API:
+        </p>
+        <ul data-link-for="PaymentRequest">
+          <li>The <a>PaymentRquest</a>'s <a>show()</a> method needs to
+          <a>triggered by user activation</a>. This reduces content's ability
```

Comment:
Should we point to https://html.spec.whatwg.org/multipage/interaction.html#activation here?